### DISCUSSION TITLE
Export data

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6118] [Q&A]
How to export data from any table as excel or csv file ?

**Comment by afonic** [comment]
There is no build in data exporter.

There are a couple of addons:
https://statamic.com/addons/steadfastcollective/csv-exporter
https://statamic.com/addons/roorda-ict/entries-export

If you are using the data, using an application like TablePlus is easier:
https://statamic.com/addons/roorda-ict/entries-export

**Reply by HazemMohamed252** [reply]
Thanks bro
